libs/libskynet/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+	"printWidth": 120,
+	"useTabs": true,
+	"semi": false
```

Comment:
I don't think there's any value in skipping semicolons while there are benefits of keeping them. 

https://prettier.io/docs/en/rationale.html#semicolons

Reply:
I like `semi: false` here. I was against it a few years back, but after switching I know that code is much more readable without them.

The benefit referenced is also a very uncommon case and, to be honest, I think we'd all rather have that array assigned to a named constant rather than just hang there for no reason.